README.md

```diff
@@ -64,7 +56,7 @@ cd lisk-node
     ```
     or, with `op-reth`:
     ```sh
-    CLIENT=reth RETH_BUILD_PROFILE=release docker compose up --build --detach
+    CLIENT=reth RETH_BUILD_PROFILE=maxperf docker compose up --build --detach
```

Comment:
@sameersubudhi 
If `maxperf` is a default profile, does it need to be present inside this command?

Reply:
Not necessary, but we just left it explicitly so that the users are aware of the same.